message_test.go

```diff
@@ -292,6 +292,27 @@ func TestRedisResult(t *testing.T) {
 		}
 	})
 
+	t.Run("AsBoolSlice", func(t *testing.T) {
+		if _, err := (RedisResult{err: errors.New("other")}).AsBoolSlice(); err == nil {
+			t.Fatal("AsBoolSlice not failed as expected")
+		}
+		if _, err := (RedisResult{val: RedisMessage{typ: '-'}}).AsBoolSlice(); err == nil {
+			t.Fatal("AsBoolSlice not failed as expected")
+		}
+		values := []RedisMessage{{string: "true", typ: '+'}}
+		if ret, _ := (RedisResult{val: RedisMessage{typ: '*', values: values}}).AsBoolSlice(); !reflect.DeepEqual(ret, []bool{true}) {
+			t.Fatal("AsBoolSlice not get value as expected")
+		}
+		values = []RedisMessage{{string: "false", typ: '+'}}
+		if ret, _ := (RedisResult{val: RedisMessage{typ: '*', values: values}}).AsBoolSlice(); !reflect.DeepEqual(ret, []bool{false}) {
+			t.Fatal("AsBoolSlice not get value as expected")
+		}
```

Comment:
true/false strings are good use cases. Could you make `AsBool()` support them as well?